AsyncDisplayKit/ASImageNode.mm

```diff
@@ -221,8 +214,8 @@ - (UIImage *)displayWithParameters:(id<NSObject> *)parameter isCancelled:(asdisp
   CGRect cropRect               = CGRectZero;
   asimagenode_modification_block_t imageModificationBlock;
 
+  ASDN::MutexLocker l(_propertyLock);
   {
-    ASDN::MutexLocker l(_imageLock);
```

Comment:
This is due to a recent change on master conflicting with the version John modified.  Let's put it back inside these braces, but still _propertyLock.